NuRadioReco/framework/parameters.py

```diff
@@ -62,5 +65,27 @@ class ARIANNAParameters(Enum):  # this class stores parameters specific to the A
     comm_period = 4  # length of data taking window
     comm_duration = 5  # maximum diration of communication window
     trigger_thresholds = 6  # trigger thresholds converted to voltage
-    l1_supression_value = 7 # This provieds the L1 supression value for given event
+    l1_supression_value = 7  # This provieds the L1 supression value for given event
     internal_clock_time = 8 # time since last trigger with ms precision
+
+
+class showerParameters(Enum):
+    zenith = 1  # zenith angle of the shower axis pointing towards xmax
+    azimuth = 2  # azimuth angle of the shower axis pointing towards xmax
+    core = 3  # position of the intersection between shower axis and an observer plane
+    energy = 4  # total energy of the primary particle
+    electromagnetic_energy = 5  # energy of the electromagnetic shower component
+    radiation_energy = 6  # totally emitted radiation energy
+    electromagnetic_radiation_energy = 7  # radiation energy originated from the electromagnetic emission
+    primary_particle = 8  # particle id of the primary particle
+    shower_maximum = 9  # position of shower maximum in slant depth, e.g., Xmax
+    distance_shower_maximum_geometric = 10  # distance to xmax in meter
```

Comment:
How are these properly defined? Wouldn't that depend on the station position?

Reply:
Yes, depends. we would define it along the shower axis from xmax to the core (intersection of shower axis with an observer plane)